Framework/Crystal/src/SCDCalibratePanels2.cpp

```diff
@@ -545,21 +558,40 @@ void SCDCalibratePanels2::optimizeBanks(IPeaksWorkspace_sptr pws, IPeaksWorkspac
     fitBank_alg->setProperty("Function", std::dynamic_pointer_cast<IFunction>(objf));
 
     //---- bounds&constraints def
-    double searchRadiusRot = getProperty("SearchradiusRotBank");
-    searchRadiusRot = std::abs(searchRadiusRot);
+    //
+    double searchRadiusRotX = getProperty("SearchradiusRotXBank");
+    searchRadiusRotX = std::abs(searchRadiusRotX);
```

Comment:
The bounded validator is already in place.
The `std::abs` here is more of a safe-guard in case someone decided to call this function outside the algorithm (which is possible btw).

Reply:
I had forgotten about the validator :slightly_smiling_face:   
`searchRadiusRotX` is initialized from property `"SearchradiusRotXBank"` at line 562. I don't know how you can set a property while at the same time bypassing its validation. At any rate, I'll mark this as resolved because it doesn't harm.